grc/sidekiq_sidekiq_tx.block.yml

```diff
@@ -1,10 +1,10 @@
 id: sidekiq_tx
-label: Sidekiq TX Source
+label: Sidekiq TX Sink
 category: '[sidekiq]'
 
 templates:
   imports: import sidekiq
-  make: sidekiq.sidekiq_tx(${sample_rate}, ${attenuation}, ${frequency}, ${bandwidth}, ${sync}, ${suppress_tune_transients}, ${dataflow_mode}, ${buffer_size} )
+  make: sidekiq.sidekiq_tx(${card}, ${port_id}, 100, ${sample_rate}, ${attenuation}, ${frequency}, ${bandwidth}, ${sync}, ${suppress_tune_transients}, ${dataflow_mode}, ${buffer_size} )
```

Comment:
I wonder if it makes sense to use the term "port_id" or if we want to use the "handle" that is typically used in libsidekiq.

Reply:
I will change

Reply:
Modified both rx and tx yml to use Handle instead of portid